Makefile

```diff
@@ -1,9 +1,12 @@
 SHELL=/bin/bash
+GIT_REF=$(shell sh -c "git rev-parse --short HEAD")
+export TOWER_OPENSHIFT_PROJECT=tower-e2e-${GIT_REF}
+
+.PHONY test/e2e/prow:
+test/e2e/prow:
+	export TOWER_OPENSHIFT_PROJECT=tower-e2e-${OPENSHIFT_BUILD_NAMESPACE}
+	make test/e2e
```

Comment:
Might want to consider doing it like we have dene in other Makefiles and just have it depend on the other target but override the env vars. https://github.com/integr8ly/integreatly-operator/blob/master/Makefile#L107-L109

Reply:
I've address that in another [pr](https://github.com/integr8ly/ansible-tower-configuration/pull/148)